source/dataman/CLS/CLSDbUpgrade1Pt2.cpp

```diff
@@ -87,9 +87,15 @@ bool CLSDbUpgrade1Pt2::upgradeImplementation()
 			iconFileName = queryFacility.value(1).toString();
 
 		} else {
-			AMErrorMon::alert(this, CLSDbUpgrade1Pt2_FAIL_TO_LOAD_FACILITY, QString("Failed to load the facility from db for %1.").arg(targetFacilityName_));
-			databaseToUpgrade_->rollbackTransaction();
-			return false;
+			// by some unknown reason, the CLSUpgrade1Pt1 didn't add the facility. readd it.
```

Comment:
@dretrex I don't think it will run on any databases at this moment, since literally, it is broken on all the beamlines other than SXRMB.